tests/unit/IOUUtilsTest.ts

```diff
@@ -25,34 +27,35 @@ describe('IOUUtils', () => {
         });
 
         test('Requesting money offline in a different currency will show the pending conversion message', () => {
-            const iouReport = ReportUtils.buildOptimisticIOUReport(1, 2, 100, 1, 'USD');
+            const iouReport = ReportUtils.buildOptimisticIOUReport(1, 2, 100, '1', 'USD');
             const usdPendingTransaction = TransactionUtils.buildOptimisticTransaction(100, 'USD', iouReport.reportID);
             const aedPendingTransaction = TransactionUtils.buildOptimisticTransaction(100, 'AED', iouReport.reportID);
+            const MergeQueries: Record<`${typeof ONYXKEYS.COLLECTION.TRANSACTION}${string}`, NullishDeep<Transaction>> = {};
```

Comment:
Maybe you can use [CollectionDataSet](https://github.com/Expensify/App/blob/main/src/types/utils/CollectionDataSet.ts) type here